Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerIISInformation.ps1

```diff
@@ -78,6 +78,7 @@ function Invoke-AnalyzerIISInformation {
     Write-Verbose "Working on IIS Web Sites"
     $outputObjectDisplayValue = New-Object System.Collections.Generic.List[object]
     $problemCertList = New-Object System.Collections.Generic.List[string]
+    $exchangeBackEndBindings = @()
```

Comment:
Can we use a List<T> here instead of an array?

Reply:
Yeah, probably should being that the other lists above are that way. I was going to say, there is only going to be a few items at most in this list so there really isn't a performance benefit, but it will make more sense if we do it that way based off the other code.